### PR TITLE
Indent prm files in examples folder and documentation

### PR DESCRIPTION
**Pull request by lpsaavedra**
# Description of the problem

- The indentation of all the parameter files in the examples folder was not uniform.
- The extracts of the parameter file in each example in the documentation was not properly indented. 

# Description of the solution

- The prmindent file located in the utilities folder was used to indent all parameter files in the examples folder.
- Tabs were removed and proper indentation was applied to the parameter file sections of each example in the documentation.

# Documentation

- A warning was added in the "contributing" tab of the documentation explaining how to properly indent sections of a parameter file when writing a new example. 

# Comments
- While fixing the indentation of the examples I noticed several differences in some examples between the parameter files explained in the documentation and the actual parameter files in the examples folder. This should be fixed to avoid confusion.

### CODE REVIEW

File: examples/unresolved-cfd-dem/boycott-effect/boycott-effect.prm [hunk 1]
```diff
@@ -199,7 +199,7 @@ subsection boundary conditions
 
   subsection lagrangian physical properties
     set gx                       = 0.0
-    set gy                       = -9.81
+    set gy                       = 0.0
```
Comment (blaisb) [inline examples/unresolved-cfd-dem/boycott-effect/boycott-effect.prm:202]: I am not sure about this, I think the gravity should be on here and not off no? This is the CFD-DEM simulation no?

**Review by blaisb — CHANGES_REQUESTED** [review]

File: examples/unresolved-cfd-dem/boycott-effect/boycott-effect.prm (hunk 1 above)
Comment (lpsaavedra) [reply examples/unresolved-cfd-dem/boycott-effect/boycott-effect.prm]: True.  I wanted to change the documentation not the actual prm. Just fixed it.

**Review by blaisb — CHANGES_REQUESTED** [review]
One tiny comment, I Have 25 files remaining

**Review by blaisb — APPROVED** [review]
Other than my two comments its good to go.
Amazing work, thank you so much!

**Comment by blaisb** [comment]
@lpsaavedra ping me once the two changes are made. I will gladly merge
:)

File: doc/source/examples/unresolved-cfd-dem/boycott-effect/boycott-effect.rst [hunk 2]
```diff
@@ -104,26 +104,26 @@ The gravity is set to 0 as we only need to insert the particles in the specified
...
-                set friction coefficient particles    = 0.3
-                set rolling friction particles        = 0.1
-        end
-        set young modulus wall           = 1e6
-        set poisson ratio wall           = 0.25
-        set restitution coefficient wall = 0.97
-        set friction coefficient wall    = 0.3
-        set rolling friction wall        = 0.1
+      set gx                       = 0.0
+      set gy                       = -9.81
```
[the remark this answers is not in the document]
Comment (lpsaavedra) [reply doc/source/examples/unresolved-cfd-dem/boycott-effect/boycott-effect.rst]: Fixed in my last commit

**Comment by lpsaavedra** [comment]
@blaisb thanks for reviewing... I know they were too many files. Both changes were addressed in the last commit and it should be ready now.

**Pull request merged**